src/test/java/com/devonfw/sample/archunit/ThirdPartyRulesE1TransactionalTest.java

```diff
@@ -0,0 +1,57 @@
+package com.devonfw.sample.archunit;
+
+import com.tngtech.archunit.core.domain.Dependency;
+import com.tngtech.archunit.core.domain.JavaClass;
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.junit.AnalyzeClasses;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.lang.ArchCondition;
+import com.tngtech.archunit.lang.ArchRule;
+import com.tngtech.archunit.lang.ConditionEvents;
+import com.tngtech.archunit.lang.SimpleConditionEvent;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.noClasses;
+
+@AnalyzeClasses(packages = "com.devonfw.sample.archunit", importOptions = ImportOption.DoNotIncludeTests.class)
+public class ThirdPartyRulesE1TransactionalTest {
+
+    private static boolean apiScopedClassIsUsingTransactional(JavaClass item, String targetPackageFullName) {
+        if(item.getFullName().contains("api") && targetPackageFullName.equals("javax.transaction.Transactional")) {
+                return true;
+        }
```

Comment:
Actually this is from the original sonar plugin rule.
We have decided to go for the modern and simplified packaging and project structure:
https://github.com/devonfw/devon4j/blob/master/documentation/guide-structure-modern.asciidoc
whereas the sonar plugin rules were based on the legacy packaging and project structure:
https://github.com/devonfw/devon4j/blob/master/documentation/guide-structure-classic.asciidoc

so before we had e.g. and interface `...logic.api.UcFindTask` and its implementation `...logic.impl.UcFindTaskImpl` whereas we now simplified and only have an implementation `...logic.UcFindTask` and its public methods are also its API. You will not find any `api` package in our sample app hence we should remove all scope related aspects here as they will not make sense and rather cause confusion.

Reply:
Fine for now. Can later be reworked with `StructuredPackage`.